json-rpc-core/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id "java"
+}
+
+group 'com.amazonaws.blox'
+version '0.1-SNAPSHOT'
```

Comment:
Should `group` and `version` come from top gradle settings?

Reply:
I think so; this is intellij autogenerated stuff though. I'll go through and clean this up uniformly after this PR.